### PR TITLE
vdk-control-cli: enable setting many options at once with vdk deploy

### PR DESCRIPTION
**Pull request by tozka**
vdk deploy --update should be able to set more than one option at once -
e.g you may want to enable and vdk version at same time in single atomic
operation. For example this is necessary in vdk version to ensure next
run uses the specified version.

We also want to keep `vdk deploy --enable` as well as `vdk deploy --update --enable --vdk-version` 

Testing Done: unit tests

Signed-off-by: Antoni Ivanov <aivanov@vmware.com>`

### CODE REVIEW

File: projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py [hunk 1]
```diff
@@ -154,13 +155,18 @@ def update(
...
+
         if job_version:
             self.__update_job_version(name, team, deployment, output)
-        elif vdk_version:
+        elif vdk_version or enabled is not None:
             self.__update_deployment(name, team, deployment)
-            log.info(
-                f"Deployment of Data Job {name} updated to use vdk version {vdk_version}."
-            )
+            msg = f"Deployment of Data Job {name} updated: "
```
Comment (doks5) [inline projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py:164]: This message seems a bit confusing. Maybe could be re-written to something like, `Data Job deployment updated; Status: `?

**Review by doks5 — CHANGES_REQUESTED** [review]

File: projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py [hunk 2]
```diff
@@ -163,27 +164,21 @@ def deploy(
...
     output: str,
 ):
     cmd = JobDeploy(rest_api_url, output)
-    if operation == DeployOperation.UPDATE:
+    if operation == DeployOperation.UPDATE or enabled is not None:
         name = get_or_prompt("Job Name", name)
         team = get_or_prompt("Job Team", team)
-        if not vdk_version and not job_version:
+        # default to ask for job version to update
+        if not vdk_version and not job_version and enabled is None:
```
[the remark this answers is not in the document]
Comment (tozka) [reply projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py:179]: We want to default to ask for job version to update if non is explicitly set. 

E.g 
```
vdk deploy --update
```
And CLI prompts for Job Version 

But if user has set something 
```
vdk deploy --update --vdk-version xx
```
or 
```
vdk deploy --update --enable
``` 
Then we do not prompt for anything and simply update based on the passed options .

File: projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py (hunk 1 above)
Comment (tozka) [reply projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py:164]:
```
Deployment of Data Job xxx updated: vdk version: 1.3; enabled; 
```

Comment (doks5) [reply projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py:164]: For consistency's sake, maybe replace the `:` after 'updated' with a `;`, and add 'status: ' before 'enabled'.

Also, `nit:` on line 168 below, there is NO `;` after "enabled", while there is one after "disabled".

File: projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (doks5) [reply projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py:179]: I see. Ok.

**Review by doks5 — APPROVED** [review]

**Pull request merged**